### DISCUSSION TITLE
flutter_service_worker.js not found

### DISCUSSION THREAD

**Discussion by naoki-sakamoto2024** [zauberzeug/nicegui#3040] [Q&A]
### Question

NiceGUIを実行したところ以下のようなログが出力されます。
このログが出ないようにするにはどうすれば良いでしょうか？

```sh
$ python xxx.py
http://127.0.0.1:8080/flutter_service_worker.js?v=3467590894 not found
http://127.0.0.1:8080/flutter_service_worker.js?v=3467590894 not found
http://127.0.0.1:8080/flutter_service_worker.js?v=3467590894 not found
```

**Comment by falkoschindler** [comment]
Hi @naoki-sakamoto2024,

1. Please use English language when posting to this repository. Thanks! ([My English is not so good, what can I do?](https://github.com/zauberzeug/nicegui/wiki/FAQs#my-english-is-not-so-good-what-can-i-do))
2. flutter_service_worker.js has nothing to do with NiceGUI itself, so the request must come from user code or another process. Some device in your network is requesting this file, which doesn't seem to be served by your app.
